src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MainConsumerImpl.java

```diff
@@ -28,6 +29,9 @@
 
     private final SlackMessagingService slackMessagingService;
 
+    @Value("${SEND_SLACK_MESSAGES}")
```

Comment:
we've got to use these naming standards for feature flags now - https://github.com/companieshouse/styleguides/blob/master/feature_flag.md 
I think we've used a prefix of FEATURE_FLAG_ before